### ISSUE TITLE
Issue installing after running npm install

### ISSUE THREAD

**Issue by albertpak** [joliss/broccoli-sass#48]
```shell
> node-sass@1.2.3 install /path_to_app/node_modules/broccoli-sass/node_modules/node-sass
> node scripts/install.js

Binary downloaded and installed at /path_to_app/node_modules/broccoli-sass/node_modules/node-sass/vendor/darwin-x64/binding.node

> node-sass@1.2.3 postinstall /path_to_app/node_modules/broccoli-sass/node_modules/node-sass
> node scripts/build.js

`darwin-x64` exists; testing
module.js:355
  Module._extensions[extension](this, filename);
                               ^
Error: Module did not self-register.
    at Error (native)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/path_to_app/node_modules/broccoli-sass/node_modules/node-sass/lib/index.js:181:15)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
npm ERR! Darwin 13.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "--save-dev" "broccoli-sass"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! node-sass@1.2.3 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@1.2.3 postinstall script 'node scripts/build.js'.
npm ERR! This is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls node-sass
```

**Comment by Ghostavio** [comment]
same here, this is due the node-sass update

**Comment by albertpak** [comment]
yeah, i had to rollback to node v0.10.35...not sure what other versions it might work with...but with latest v0.12.0, it definitely doesn't

**Comment by aexmachina** [comment]
See #43.

**Comment by albertpak** [comment]
@xzyfer This might be related to issue [#490](https://github.com/sass/node-sass/issues/490#issuecomment-74353387) for node-sass

**Comment by xzyfer** [comment]
@albertpak correct node-sass < 2.0.0 is not compatible with node 0.12. Either downgrade to 0.10 (or use nvm),  or wait til https://github.com/joliss/broccoli-sass/pull/43 is merged as @exmachina points out.

**Comment by aexmachina** [comment]
Resolved by #47.
